zappa/core.py

```diff
@@ -2422,11 +2434,23 @@ def get_api_id(self, lambda_name):
         except Exception:  # pragma: no cover
             try:
                 # Try the old method (project was probably made on an older, non CF version)
-                response = self.apigateway_client.get_rest_apis(limit=500)
+                rest_apis_list = []
```

Comment:
@souravjamwal77
Boto3 has a max limit of 500 per get_rest_apis call. I think it's better to generalize the solution instead of hardcoding the value, so we don't have to change it in case AWS increases these limits in the future.

Reply:
Yes, according to the boto3 docs the `get_rest_apis()` max is 500 and default is 25.

https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/apigateway.html#APIGateway.Client.get_rest_apis

I don't think it's an issue if we set to the max, 500, do you?